tests/test_reference_es_mapping_modifier.py

```diff
@@ -0,0 +1,70 @@
+from django.test import TestCase
+from unittest.mock import MagicMock, patch
+
+from arches_controlled_lists.search.references_es_mapping_modifier import (
+    ReferencesEsMappingModifier,
+)
+
+
+class TestReferencesEsMappingModifier(TestCase):
+    @patch("arches_controlled_lists.models.ListItem.objects.get")
```

Comment:
This mock doesn't seem essential, and in fact, removing it shows an interesting failure:

```py
django.core.exceptions.ValidationError: ['“test_value” is not a valid UUID.']
```

I think it would be better to use a UUID so the test documents real usage. Plus what if we change the `get()` to `first()` in future etc

***
If you want need to factor out a test object factory util from the other test class instead of copy-pasting it here, that's fine with me.

Reply:
done!